server/cmd/mattermost/commands/export.go

```diff
@@ -134,11 +136,10 @@ func scheduleExportCmdF(command *cobra.Command, args []string) error {
 		ctx := context.Background()
 		if timeoutSeconds > 0 {
 			var cancel context.CancelFunc
-			ctx, cancel = context.WithTimeout(ctx, time.Second*time.Duration(timeoutSeconds))
+			_, cancel = context.WithTimeout(ctx, time.Second*time.Duration(timeoutSeconds))
```

Comment:
We still need the `ctx` that `context.WithTimeout`returns, right?

Reply:
yes that is true I forgot to assign it back

Reply:
I've done it